### PR TITLE
Added jump search in Searching Algorithms

### PR DESCRIPTION
**Pull request by Sumangrewal**
I have added Jump Search Algorithm

### CODE REVIEW

File: Algorithms/Searching/jumpsearch.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// C++ program to implement Jump Search 
+
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/jumpsearch.cpp:2]: please mention sample input output in comments

File: Algorithms/Searching/jumpsearch.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  
+    return -1; 
+} 
+  
+// Driver program to test function 
+int main() 
+{ 
+    int arr[] = { 0, 1, 1, 2, 3, 5, 8, 13, 21, 
+                34, 55, 89, 144, 233, 377, 610 }; 
+    int x = 55;
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/jumpsearch.cpp:45]: avoid single letter variables

File: Algorithms/Searching/jumpsearch.cpp [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    return -1; 
+} 
+  
+// Driver program to test function 
+int main() 
+{ 
+    int arr[] = { 0, 1, 1, 2, 3, 5, 8, 13, 21, 
+                34, 55, 89, 144, 233, 377, 610 }; 
+    int x = 55; 
+    int n = sizeof(arr) / sizeof(arr[0]);
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/jumpsearch.cpp:46]: here too

File: Algorithms/Searching/jumpsearch.cpp (hunk 2 above)
Comment (Sumangrewal) [reply Algorithms/Searching/jumpsearch.cpp:45]: Yes I have changed it now.

File: Algorithms/Searching/jumpsearch.cpp (hunk 3 above)
Comment (Sumangrewal) [reply Algorithms/Searching/jumpsearch.cpp:46]: This too.

**Comment by Sumangrewal** [comment]
3afdbdc This is the updated Algorithm for Jump Search

File: Algorithms/Searching/binarysearch.cpp [hunk 4]
```diff
@@ -27,13 +27,18 @@ int binarySearch(int arr[], int l, int r, int x)
     // not present 
     return -1; 
 } 
+  // Sample input:
```
Comment (Bhavnaharitsa) [inline Algorithms/Searching/binarysearch.cpp]: push the sample input output on top under the description

Comment (Sumangrewal) [reply Algorithms/Searching/binarysearch.cpp]: Done.

**Comment by Bhavnaharitsa** [comment]
Your points will be updated soon

**Pull request merged**